ugs-core/src/com/willwinder/universalgcodesender/MacroHelper.java

```diff
@@ -95,22 +95,22 @@ public static void executeCustomGcode(final String str, BackendAPI backend) thro
      * @return 
      */
     protected static String substituteValues(String str, BackendAPI backend) {
-        SystemStateBean bean = new SystemStateBean();
```

Comment:
We don't rely on the system state bean anymore. That logic is now moved to the pendant implementation.